lib/node_modules/@stdlib/time/base/parse-duration/README.md

```diff
@@ -0,0 +1,115 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2022 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Parse Duration
+
+> Parse a duration string into an object with `minutes`, `seconds`, and `milliseconds`.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var parseDuration = require( '@stdlib/time/base/parse-duration' );
+```
+
+#### parseDuration
+
+Parses a duration string such as `1m3s10ms` into an object with `minutes`, `seconds`, and `milliseconds`.
+
+```javascript
+var obj = parseDuration( '1m3s10ms' );
+// returns { 'minutes': 1, 'seconds': 3, 'milliseconds': 10 }
+
+obj = parseDuration( '1m3s' );
+// returns { 'minutes': 1, 'seconds': 3, 'milliseconds': 0 }
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   A duration string is a string containing a sequence of time units. A time unit is a non-negative integer followed by a unit identifier. The following unit identifiers are supported:
```

Comment:
IMO we should also support `h` (hours) and `d` (days). Supporting `y` (years) would be harder given the fact that a year is not a sixed amount of time.

Reply:
Was leaning that way too. Will update!

Reply:
Changed accordingly.